core/test/scripts/PublishPrices.js

```diff
@@ -0,0 +1,41 @@
+const PublishPrices = require("../../scripts/PublishPrices.js");
+const assertPackage = require("assert");
+
+contract("scripts/PublishPrices.js", function(accounts) {
+  it("verifyFeedConfig", async function() {
+    const validConfig = {
```

Comment:
What about also testing the committed config file(s) under core? (You could also use that to replace this inline config if that would be easier)

Reply:
Done. I'd rather use a separate config for the tests, so that we can be sure that it (1) has a denominator, and (2) at least one of the values is "0" (to make sure that's valid, I can never tell with javascript).